src/modules/blocks/attendees/template.js

```diff
@@ -82,27 +88,80 @@ const RenderSubtitle = () => (
 	</div>
 );
 
-const Attendees = ( props ) => {
-
-	const { isSelected, title } = props;
+const UI = ( props ) => {
+	const { isSelected, title, displayTitle, displaySubtitle } = props;
 	const blockTitle = ! ( isSelected || title )
```

Comment:
I'm a little bit confused by the logic here. Is the `isSelected` actually necessary here? If there is no title, then we show a placeholder. If there is a title, we show the title in the input. Are we showing the placeholder if the block is not selected?

Reply:
Yes, we're showing a default value for the title. This default value is the same that the block has on the FE.